### ISSUE TITLE
accountID and carrier is not getting tracked for AppleTV

### ISSUE THREAD

**Issue by samsonroyquickplay** [newrelic/newrelic-react-native-agent#96]
@ndesai-newrelic We have patched the new relic to support AppleTV by changing in the podspec file. But somehow carrier and accountID is not getting tracked by new-relic SDK. Which are deffault attributes which get tracked by new-relic ? Any suggestions on how to solve this ?

Tech Stack: ReactNative
New RelicVersion: 1.0.1

**Comment by ndesai-newrelic** [comment]
@samsonroyquickplay We are not capturing carrier value but which accountID are you talking about? Can you provide us example ScreenShot ?

**Comment by samsonroyquickplay** [comment]
@ndesai-newrelic in mobiles we see carrier having values 'wifi' . Is that not applicable for TV ?

**Comment by ndesai-newrelic** [comment]
@samsonroyquickplay we found this issue on our side too, we will fix it soon

**Comment by samsonroyquickplay** [comment]
@ndesai-newrelic  When can we expect this release ?

**Comment by kthai-nr** [comment]
@samsonroyquickplay We are hoping to have a release with this fix for tvOS out by the end of the month. Once it releases, we can notify you via this thread.

**Comment by mbruin-NR** [comment]
@samsonroyquickplay we were able to fix a bug where the connection type was being reported as unknown in http events recorded by New Relic in the newest iOS agent release. However I don't think that answers what you are looking for. After talking it over more with the team we decided that Apple tv's can't have carriers like phones do so that metric just doesn't apply. Sorry for the confusion but there are no plans to change this at this time.
